ocp_resources/cluster_role.py

```diff
@@ -1,51 +1,31 @@
 # -*- coding: utf-8 -*-
-from ocp_resources.constants import TIMEOUT_4MINUTES
 from ocp_resources.resource import Resource
 
 
 class ClusterRole(Resource):
     """
     ClusterRole object
+    'rbac_authorization_k8s_io' API official docs:
+        https://docs.openshift.com/container-platform/3.11/rest_api/rbac_authorization_k8s_io/rbac-authorization-k8s-io-index.html
```

Comment:
please update to the api - https://kubernetes.io/docs/reference/kubernetes-api/authorization-resources/cluster-role-v1/